src/trimEnd/trimEnd.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { trimEnd } from './trimEnd';
```

Comment:
If all individual utility functions would get their own folder, we would quickly lose the overview.
I rather bundle them in a folder (like we have for internal hooks), or find another way to segregate the exposed hooks from our internal utilities.

Reply:
I agree, created a utils folder and added the trimEnd function to that folder 👍🏻